### DISCUSSION TITLE
Easier way for new users to set Starting 1RM values for all Exercises

### DISCUSSION THREAD

**Discussion by Bruno-366** [astashov/liftosaur#214] [Ideas]
I suggest prompting a new user to input **body weight** and **experience** level.
Then based on that input adjust the starting weights for all exercises, rather than a user having to go through the entire list of exercises (which can be quite varied in some programs) and update the values one-by-one.

For example [Strength Level](https://strengthlevel.com/strength-standards/male/kg) has the following experience levels:

- Beginner
- Novice 	
- Intermediate
- Advanced
- Elite

While  [Symmetric Strength](https://symmetricstrength.com/standards#/) has the following experience levels:

- Untrained 	
- Novice 	
- Intermediate 	
- Proficient 	
- Advanced 	
- Exceptional 	
- Elite 	
- World class

**Comment by astashov** [comment]
This is done - when you pick a program that uses 1RM, it offers you to fill in 1RM for exercises used in the program
